packages/client/src/__tests__/integration/happy/signals/test.ts

```diff
@@ -22,7 +22,7 @@ async function waitMessageOnStdout(child: ExecaChildProcess): Promise<string> {
   return message.toString().trim()
 }
 
-describe('signals that should terminate the process', () => {
+describe.skip('signals that should terminate the process', () => {
```

Comment:
TODO: try to re-enable this, last time this had isues in swc.

Reply:
I think that adding a comment in the file would be better here

Reply:
Re-enabled.